packages/land/package.json

```diff
@@ -1,10 +1,15 @@
 {
   "name": "@sandbox-smart-contracts/land",
-  "version": "0.0.1",
+  "version": "0.0.3",
```

Comment:
Why 0.0.3? Did we release a package yet?

Reply:
As I said I had some issues with release-it